src/malloy/data/bigquery/bq_connection.py

```diff
@@ -80,8 +80,6 @@ def get_schema_for_sql_block(self, name, sql):
             "type": "basetable",
             "connectionName": self.get_name(),
         },
-        # TODO: Fix protected-access when alternative available
```

Comment:
Pretty sure this is still needed unless they've fixed access to the schema in the normal api, and then we'd need to update the access here.